consensus/quorum/quorum.go

```diff
@@ -125,13 +124,13 @@ type Decider interface {
 
 // Registry ..
 type Registry struct {
-	Deciders      map[uint32]Decider `json:"quorum-deciders"`
+	Deciders      map[string]Decider `json:"quorum-deciders"`
```

Comment:
why change from uint32 to string? It's harder to address the key now.

Reply:
This was only used for super committee RPC and makes key easier to pull for downstream consumer watchdog